app/views/devise/confirmations/new.html.erb

```diff
@@ -4,13 +4,25 @@
   <%= render "devise/shared/error_messages", resource: resource %>
 
   <div class="field">
-    <%= f.label :email %><br />
-    <%= f.email_field :email, autofocus: true, autocomplete: "email", value: (resource.pending_reconfirmation? ? resource.unconfirmed_email : resource.email) %>
+    <%= f.label :email %><br/>
+    <%= f.email_field :email,
```

Comment:
Since we are doing enhancements to the sign in page and have touched the devise templates, can I get you to add translations for the labels, buttons (submit, etc;), and any other elements that are missing translations, to the templates under `passwords/`, `registrations/`, `unlocks/`

Reply:
Assuming I can ignore the mailer templates, please see d79b221721c5aac23e23772fa75a825607a41b53.

Reply:
We may as well add translations for the mailer templates as well. We will need them at some point!

Reply:
Please see f721f8e35db852a440823cf56b9db8deb8ff094e.